tests/test_maplestory.py

```
@@ -3,33 +1,122 @@
+                    "name": "綠髮帶",
+                    "type": "裝備",
+                    "link": "https://maplesaga.com/library/cn/permalink/equip/1002067",
+                    "img": "https://mai0313.github.io/artale-drop/image/%E7%B6%A0%E9%AB%AE%E5%B8%B6.png",
+                }
+            ],
+        }
+    ]
+
+
+@patch.object(MapleStoryCogs, "_load_monsters_data")
+def test_load_monsters_data_with_patch(mock_load_data: Mock) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)